doc/btp_pacs.txt

```diff
@@ -0,0 +1,30 @@
+Published Audio Capabilities Service (ID 12)
+=============================
+
+Commands and responses:
+
+	Opcode 0x00 - Error response
+
+	Opcode 0x01 - Read supported commands
+		Controller Index:    <controller id>
+		Command parameters:  <none>
+		Response parameters: <supported commands> (variable)
+
+		Each bit in response is a flag indicating if command with
+		opcode matching bit number is supported. Bit set to 1 means
+		that command is supported. Bit 0 is reserved and shall always
+		be set to 0. If specific bit is not present in response (less
+		than required bytes received) it shall be assumed that command
+		is not supported.
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x02 - Update characteristic
+		Controller Index:    <controller id>
+		Command parameters:  ID of the characteristic (1 octet)
```

Comment:
add allowed values for ID

Reply:
For the current  PACS.TS.p0, it looks like any change is sufficient. I'm wondering if there should be specific commands like Update Source PAC, with an explicit PAC Record. Or should we wait until a PTS requires it? The BAP tests don't require that though.

Reply:
we can extend this (or add new dedicated command) if needed
those values here are for BTP,  IUT doesn't have to implement all of them

Reply:
I've extended the command.